src/overlays/actors/ovl_En_Md/z_en_md.c

```diff
@@ -654,7 +654,7 @@ void EnMd_Init(Actor* thisx, PlayState* play) {
     s32 pad;
 
     ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 24.0f);
-    SkelAnime_InitFlex(play, &this->skelAnime, &gMidoSkel, NULL, this->jointTable, this->morphTable, 17);
+    SkelAnime_InitFlex(play, &this->skelAnime, &gMidoSkel, NULL, this->jointTable, this->morphTable, ENMD_LIMB_MAX);
```

Comment:
I'm not sure if this 17 should be `ENMD_LIMB_MAX` or `ARRAY_COUNT(this->jointTable)` / `ARRAY_COUNT(this->morphTable)`


(same with `func_80034F54`)

I think I'm myself OK with using the `_MAX` enum value in these cases

Anyone else got an opinion?

Reply:
we've used _MAX so far fwiw. but yea i guess you can technically use array count here. Am partial to _MAX cause its what im used to, but otherwise no opinon